packaging/nuget/NServiceBus.Metrics.PerformanceCounters.nuspec

```diff
@@ -14,8 +14,8 @@
     <copyright>$copyright$</copyright>
     <tags>$tags$</tags>
     <dependencies>
-      <dependency id="NServiceBus" version="[6.2.0, 7.0.0)" />
-      <dependency id="NServiceBus.Metrics" version="[1.1.0, 2.0.0)" />
+      <dependency id="NServiceBus" version="[7.0.0-*, 8.0.0)" />
+      <dependency id="NServiceBus.Metrics" version="[2.0.0-*, 3.0.0)" />
```

Comment:
Is there a reason to stay with the old packaging?

Reply:
no. but i wasnt sure how much to scope in this PR. so decided to leave it as nsb 7 for now

Reply:
Makes sense, I have a call with @seanfarmar today to get him up to speed on the new packaging